packages/react-native-dogfood/src/screens/Call/JoinCallScreen.tsx

```diff
@@ -134,6 +134,7 @@ const JoinCallScreen = () => {
           placeholder="Enter comma separated User Ids"
           style={styles.textInput}
           value={ringingUserIdsText}
+          autoCapitalize="none"
```

Comment:
unrelated change.. but we better shouldnt autocapitalise the custom user name calls